src/sentry/api/endpoints/project_details.py

```diff
@@ -23,8 +23,8 @@
 from sentry.auth.superuser import is_active_superuser
 from sentry.constants import RESERVED_PROJECT_SLUGS
 from sentry.datascrubbing import validate_pii_config_update
-from sentry.dynamic_sampling.rules_generator import generate_rules
-from sentry.dynamic_sampling.utils import get_supported_biases_ids, get_user_biases
+from sentry.dynamic_sampling.rules.base import generate_rules
```

Comment:
you can declare these functions in dynamic_sampling `__init__.py` Similar
https://github.com/getsentry/sentry/blob/9af57a33c19004dd6d8ddb54cf55baa3d91146f6/src/sentry/receivers/__init__.py#L1-L16 and avoid such changes.

Reply:
I actually did that in my first iterations but then had problems with mypy even after adding the comment.

Reply:
I will try again to implement it and see which errors I get.